data/strings/strings.txt

```diff
@@ -9847,12 +9847,82 @@
     comment = Tab title in the time picker
     tags = android
     en = From
+    ar = من الساعة
+    be = З
+    bg = От
+    ca = De
+    cs = Od
+    da = Fra
+    de = Von
+    el = Από τις
+    es = De
+    et = Kell
+    eu = etatik
+    fa = از ساعت
+    fi = alkaen
+    fr = De
+    he = משעה 6:00 עד 11:00
+    hu = tól
+    id = Dari pukul
+    it = Dalle
+    ja = から
+    ko = 부터
+    mr = ते
+    nb = Fra
+    nl = Van
     pl = Od
+    pt = Das
+    pt-BR = Das
+    ro = De la
+    ru = С
+    sk = Od
+    sv = Från
+    th = เวลา
+    tr = den itibaren
+    uk = З
+    vi = Từ
+    zh-Hans = 从
+    zh-Hant = 從
 
   [editor_time_to]
     tags = android
-    en = To
-    pl = Do
+    en = to
```

Comment:
Does lower cased `to` look better in that context? If capital one is better, then should the first letters in other langs be converted to capital ones?

Reply:
It should not matter as the picker converts them to all caps.
Do you know any tool that can convert a string to "sentence case" and do so reliably for all scripts?

Reply:
Do all caps look good?

You can select the first LTR word in Visual Studio Code and select Title Case. I would not touch RTL langs and non-latin/non-cyrillic scripts.

Reply:
The all caps were already there before all my changes. It's a Material Design convention I think.
I managed to convert strings to sentence case.